GameData/RP-1/Contracts/Commercial Applications 1/EarlyNavNetwork3.cfg

```diff
@@ -95,7 +91,7 @@ CONTRACT_TYPE
 			name = HasPower
 			type = HasResource
 			resource = ElectricCharge
-			minQuantity = 1.0
+			minQuantity = 2d
```

Comment:
That's a duration being specified for a quantity param?

Reply:
I didn't ask the right question on a previous iteration of the EC requirement and should have clarified it then.  Realized the better way was the power generation check anyway.  Even with the short time requirement, it made sense to me to keep it on those since it would be required in actuality.